core/models/vmid.go

```diff
@@ -31,7 +31,7 @@ func NewVMID(name, namespace string) (*VMID, error) {
 	}
 
 	if namespace == "" {
-		return nil, coreerrs.ErrNamespaceRequired
+		namespace = "default"
```

Comment:
Can we put store this in a constant, so other packages can use this as value and we have to update only one place?